### PR TITLE
Added test on copyVariant function

### PR DESCRIPTION
**Pull request by shanshin**

### CODE REVIEW

File: kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/ExamplesBuildTests.kt [hunk 1]
```diff
@@ -24,6 +24,11 @@ internal class ExamplesBuildTests {
         // build only
     }
 
+    @ExamplesTest("jvm/copy-variant", ["koverHtmlReportFirst", "koverHtmlReportSecond"])
+    fun CheckerContext.jvmCopyVariant() {
+        // build only
```
Comment (sandwwraith) [inline kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/ExamplesBuildTests.kt:29]: Why build only? Shouldn't you check that `first` variant covers only `SubprojectUtils` and `ClassFromSecondProject` is absent?

File: kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/appliers/FinalizeKover.kt [hunk 2]
```diff
@@ -58,13 +58,6 @@ internal fun KoverContext.finalizing(origins: AllVariantOrigins) {
     jvmVariant?.let { variantArtifacts[JVM_VARIANT_NAME] = it }
     androidVariants.forEach { variantArtifacts[it.variantName] = it }
 
-    val availableVariants = variantArtifacts.keys + projectExtension.currentProject.customVariants.keys
```
Comment (sandwwraith) [inline kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/appliers/FinalizeKover.kt:61]: Is this related to the test? I don't know why this code got deleted

Comment (shanshin) [reply kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/appliers/FinalizeKover.kt:61]: This code was accidentally added during the merging, but the error was not detected because there was no test

**Review by sandwwraith — APPROVED** [review]
